stat/distuv/gumbel.go

```diff
@@ -0,0 +1,116 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+
+	"golang.org/x/exp/rand"
+)
+
+// Gumbel implements the Gumbel distribution, a two-parameter continuous
+// distribution with support over the real numbers.
+//
+// The Gumbel distribution has density function
+//  1/beta * exp(-(z + exp(-z)))
+//  z = (x - mu)/beta
+// Beta must be greater than 0.
+//
+// For more information, see https://en.wikipedia.org/wiki/Gumbel_distribution .
```

Comment:
I think you can omit the space before the period, but please check.

Reply:
Done.